services/provider/server/server.go

```diff
@@ -1229,13 +1231,41 @@ func (s *OCSProviderServer) getDesiredCephConnection(
 		return nil, fmt.Errorf("failed to extract monitor IPs from configmap %s: %v", monConfigMap, err)
 	}
 
+	var readAffinity *csiopv1a1.ReadAffinitySpec = nil
+	readAffinityOptions := util.GetReadAffinityOptions(storageCluster)
```

Comment:
readAffinityOptions -> readAffinity

Reply:
I'm using the readAffinity variable name for another variable just above, so I kept readAffinityOptions here to avoid conflict